### PR TITLE
Made changes to FAQ

### PR DESCRIPTION
**Pull request by sangeet259**
- Updated the College Requirement
         Added a relevant [Quora Answer](https://www.quora.com/What-is-the-prerequisite-for-participating-in-Kharagpur-Winter-of-Code-2016/answer/Pranit-Bauva) by @pranitbauva1997 
- Changed one 2016 to 2017

### CODE REVIEW

**Review by pranitbauva1997 — APPROVED** [review]
LGTM! Thanks!

**Comment by harishnandan** [comment]
I don't think we need the year specification. Isn't just KWoC fine?
@pranitbauva1997  @sangeet259

File: templates/faq.html [hunk 1]
```diff
@@ -239,7 +239,7 @@ <h1 class="white-text">Hello, how can we help you?</h1>
                                 <!-- faq item -->
                                 <div class="panel panel-default">
                                     <div role="tablist" id="type1-headingFour" class="panel-heading">
-                                        <a data-toggle="collapse" data-parent="#collapse-three" href="#collapse-three-link9"><h4 class="panel-title">Will I receive a certificate at the end of the KWoC 2016?<span class="pull-right"><i class="fa fa-plus"></i></span></h4></a>
+                                        <a data-toggle="collapse" data-parent="#collapse-three" href="#collapse-three-link9"><h4 class="panel-title">Will I receive a certificate at the end of the KWoC 2017?<span class="pull-right"><i class="fa fa-plus"></i></span></h4></a>
```
Comment (pranitbauva1997) [inline templates/faq.html]: As @harishnandan points out, even I think it would be good if we just keep it at the end of `kwoc program`.

**Review by pranitbauva1997 — CHANGES_REQUESTED** [review]
Except for the @harishnandan thing, everything is good!

**Comment by sangeet259** [comment]
Done

**Review by pranitbauva1997 — APPROVED** [review]

**Pull request merged**